websites/R/Roblox/presence.ts

```diff
@@ -1,19 +1,17 @@
-var presence = new Presence({
+const presence = new Presence({
   clientId: "612416330003382314"
-});
-
-var browsingStamp = Math.floor(Date.now() / 1000);
-var profileName: any,
-  profileTabs: any,
-  messageTab: any,
-  friendsTab: any,
-  inventoryTab: any,
-  groupName: any,
-  groupTab: any,
-  gameName: any,
-  gameTab: any;
-
-gameName = document.querySelector(
+}), browsingStamp = Math.floor(Date.now() / 1000);
+let
+  profileName,
+  profileTabs,
+  messageTab,
+  friendsTab,
+  inventoryTab,
+  groupName,
+  groupTab,
+  gameTab;
+
+const gameName = <HTMLHeadingElement>document.querySelector(
   "div.game-calls-to-action > div.game-title-container > h2"
 );
```

Comment:
move this up and merge with other const, but I have a feeling this one should've been set in updatedata event?

Reply:
Yep, just checked and it's supposed to be inside updatedata event. Made the necessary changes.